UnofficialSteamAuthenticator/Models/User.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Windows.UI.Xaml.Media.Imaging;
+using SteamAuth;
+
+namespace UnofficialSteamAuthenticator.Models
+{
+    /// <summary>
+    /// Type for showing users in a list
+    /// </summary>
+    internal class User : ModelBase
+    {
+        public ulong SteamId { get; }
+        public string Title { get; set; }
+        public string Content { get; set; }
+        public BitmapImage Avatar { get; set; }
+        private readonly SteamGuardAccount _steamGuardAccount;
+
+        public User(ulong steamId, string title, string content, BitmapImage avatar)
+        {
+            SteamId = steamId;
+            _steamGuardAccount = Storage.GetSteamGuardAccount(steamId);
+
+            Title = title;
```

Comment:
Look at this and tell me to commit it... Because I'm **sure **you will regret this :D

![image](https://cloud.githubusercontent.com/assets/6497266/13879869/007d23c0-ed1a-11e5-999d-0a95d71ba4d3.png)

Not only did the capitalization help, but the underscores as well... :P